File: section3/27BigO_scalability/ex.py
import time

nemo = ['dory','bruce','martin','nemo','gill','bloat','nigel','squirt','darla','hank']

def find_nemo(array):
    # t0 = time.time()
    for i in nemo:
        if(i == 'nemo'):
            print('Found nemo')
            break
    # t1 = time.time()
    # print(t1-t0)
            
find_nemo(nemo)

File: section3/27BigO_scalability/ex.go
package main

import (
	"fmt"
	// "time"
)

func findNemo(array []string) {
	// start := time.Now();
	for i := 0; i < len(array); i++ {
		if (array[i] == "nemo") {
			fmt.Println("Found nemo");
			break;
		}
	}
	// t := time.Now()
	// elapsed := t.Sub(start);
	// fmt.Println(elapsed);
}

func main () {
	nemo := []string{"dory","bruce","martin","nemo","gill","bloat","nigel","squirt","darla","hank"};
	findNemo(nemo)
}